src/AzureExtension/DevBox/DevBoxProvider.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System.Text.Json;
+using AzureExtension.Contracts;
+using DevHomeAzureExtension.DataModel;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.Foundation;
+
+namespace AzureExtension.DevBox;
+
+public class DevBoxProvider : IComputeSystemProvider, IDisposable
+{
+    private readonly IHost _host;
+
+    public DevBoxProvider(IHost host)

```

Comment:
For this PR I'd just remove this constructor. Then in the PR where you have the code that uses the host container to do stuff, add it in again.